storage/src/create_bucket_dual_region.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/storage/README.md
+ */
+
+namespace Google\Cloud\Samples\Storage;
+
+# [START storage_create_bucket_dual_region]
+use Google\Cloud\Storage\StorageClient;
+
+/**
+ * Create a new bucket with a custom default storage class and location.
+ *
+ * @param string $bucketName The name of your Cloud Storage bucket.
+ * @param string $location1 First location for the bucket's regions. Case-insensitive.
+ * @param string $location2 Second location for the bucket's regions. Case-insensitive.
+ */
+function create_bucket_dual_region($bucketName, $location1, $location2)
+{
+    // $bucketName = 'my-bucket';
+    // $location1 = 'US-EAST1';
+    // $location2 = 'US-WEST1';
+
+    $storage = new StorageClient();
+    $bucket = $storage->createBucket($bucketName, [
+        'location' => "${location1}+${location2}",
+    ]);
+
+    printf("Created dual-region bucket '%s' in %s+%s", $bucket->name(), $location1, $location2);
```

Comment:
nit: do we want to surround the region with single quotes as we did the bucket name?
```suggestion
    printf("Created dual-region bucket '%s' in '%s+%s'", $bucket->name(), $location1, $location2);
```